frontend/src/components/regions/CardDetails/hooks/openAndCloseReviewPopUp.js

```diff
@@ -0,0 +1,9 @@
+import { useState } from "react";
+
+export default function useOpenAndCloseReviewPopUp() {
```

Comment:
This seems like you are overcomplicating things. 

Remember that the code exists for a purpose. The purpose is not to make you look clever. The purpose is to work.

Dont ever send the people reviewing your code on a wild goose chase.

Junior coders think `fancy code === good code`
Senior coders know that  `simple code === excellent code`